### DISCUSSION TITLE
Harvester Verification Issue

### DISCUSSION THREAD

**Discussion by iempey** [Chia-Network/chia-blockchain#5757] [Support]
Hello, 

I have set up a harvester on my gaming laptop and I see the connection in "Your Harvester Network" in the GUI, and I see the plots show up in the "Last attempted proof" area checking for block challenges. 

My problem is that when I check the logs after changing them to INFO, I can't find any "-> new_signage_point_harvester to peer <harvester_ip" in my logs. 

I feel like something was not set up correctly even though I see everything in the GUI. 

Should re-set up my Harvester?

Thanks-

**Comment by BrandtH22** [comment]
@iempey , if you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
